2016/playbook/session_2/SCHEDULE.md

```diff
@@ -87,12 +87,12 @@
 | 9:00 AM  | Hacker arrival                                     |
 | 9:10 AM  | Energizer 1                                        |
 | 9:25 AM  | Breakfast                                          |
-| 9:45 AM  | [Dodge][dodge]          						    |
+| 9:45 AM  | [Dodge][dodge]          						                |
 | 12:30 PM | Lunch                                              |
 | 1:30 PM  | Energizer 2                                        |
-| 2:00 PM  | [Dodge (continued)][dodge] 						|
+| 2:00 PM  | [Dodge (continued)][dodge] 						            |
 | 3:30 PM  | [Energizer 3](../ACTIVITIES.md#walking)            |
-| 3:45 PM  | [Dodge (continued)][dodge] 						|
+| 3:45 PM  | [Dodge (continued)][dodge] 					            	|
```

Comment:
Idk what's up with my table formatting, it all looks right when I do it in Sublime and in the GitHub browser editor